stable/fluent-bit/README.md

```diff
@@ -75,6 +75,9 @@ The following table lists the configurable parameters of the Fluent-Bit chart an
 | `image.fluent_bit.repository`      | Image                                      | `fluent/fluent-bit`                               |
 | `image.fluent_bit.tag`             | Image tag                                  | `0.13.0`                                          |
 | `image.pullPolicy`                 | Image pull policy                          | `Always`                                          |
+| `prometheus.operator.enabled`                  | Are you using Prometheus Operator?  [Blog Post](https://coreos.com/blog/the-prometheus-operator.html) | `true`                            |
```

Comment:
I don't believe this should be enabled by default

Reply:
Fixed